dataflow/build.gradle

```diff
@@ -54,46 +55,59 @@ createDataflowShaded('shaded')
 createDataflowShaded('nullaway')
 createDataflowShaded('errorprone')
 
-task liveVariableTest(dependsOn: [assemble, compileTestJava], group: 'Verification') {
-    description 'Test the live variable analysis test for dataflow framework.'
-    inputs.file('tests/live-variable/Expected.txt')
-    inputs.file('tests/live-variable/Test.java')
-
-    outputs.file('tests/live-variable/Out.txt')
-    outputs.file('tests/live-variable/Test.class')
-
-    delete('tests/live-variable/Out.txt')
-    delete('tests/live-variable/Test.class')
-    doLast {
-        javaexec {
-            workingDir = 'tests/live-variable'
-            if (!JavaVersion.current().java9Compatible) {
-                jvmArgs += "-Xbootclasspath/p:${configurations.javacJar.asPath}".toString()
+/**
+ * create a task to run dataflow analysis demo
+ * @param taskName specify the taskName of created task
+ * @param dirName specify the directory name of the test file
+ * @param pkgName specify the package name of the demo
+ * @param className specify the class name of the demo
+ * @return
+ */
+def testDataflowAnalysisDemo(taskName, dirName, pkgName, className) {
```

Comment:
Is the `Demo` necessary? The name already starts with `test`.

Reply:
Yes, it's a bit redundant. I will remove that.